drivers/aufs/aufs.go

```diff
@@ -347,7 +347,9 @@ func (a *Driver) createDirsFor(id, parent string) error {
 // Remove will unmount and remove the given id.
 func (a *Driver) Remove(id string) error {
 	a.locker.Lock(id)
-	defer a.locker.Unlock(id)
+	defer func(id string) {
```

Comment:
(All of these can be done without the explicit `id` parameter, although this works fine as is.)